content/docs/start/data-pipelines.md

```diff
@@ -292,17 +292,18 @@ prepare:
 DVC pipelines (`dvc.yaml` file, `dvc run`, and `dvc repro` commands) solve a few
 important problems:
 
-- _Automation_ - run sequence of steps in a "smart" way that makes iterating on
-  the project faster. It automatically determines which parts of a project need
-  to be run, it caches "runs" and results — all to avoid running the same stage
-  again.
-- _Reproducibility_ - it can describe and capture what data should be used and
-  what commands to run to produce an ML model, for example. It's described and
-  captured in way that is easy to put into Git. It means that it's easy to
-  version and share.
-- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - describing
-  project in way that it can be reproduced (built) is the fist necessary step
-  before introducing CI/CD systems.
+- _Automation_ - run a sequence of steps in a "smart" way that makes iterating on
+  your project faster. DVC automatically determines which parts of a project
+  need to be run, and it caches "runs" and their results, to avoid unnecessary
+  re-runs.
+- _Reproducibility_ - `dvc.yaml` and `dvc.lock` files describe what data to use
+  and which commands will generate the pipeline results (such as an ML
+  model). Storing these
+  files in Git makes it easy to version and share.
+- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - reproducible
+  ML pipelines allow CI/CD systems to retrain models on fresh
+  datasets with identical training, save the results, and even produce reports
+  about the whole process. See [CML.dev](https://cml.dev/) for some examples.
```

Comment:
How about this @shcheklein ?
```suggestion
- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - reproducible
  ML pipelines allow CI/CD systems to reproduce all or part of the process (e.g.
  model training) on production datasets. Using DVC, they can then save the
  results, and even produce reports about the whole process. See
  [CML.dev](https://cml.dev/) for some examples.
```

Reply:
Feel free to edit/commit it and merge the Restyled PR if you agree 🙂 or lmk and I'll do so.

Reply:
it still describes one very specific case "reproduce all or part of the process ... on production datasets"

let's go back a bit, gusy. What was the problem with initial version? So, that I understand how to help you solve it.

Reply:
It just had a small typo, Noah decided to rewrite it all, for readability I think. Here's the original version (typo fixed) in case we want to just go back to that:
```suggestion
- _Continuous Delivery and Continuous Integration (CI/CD) for ML_ - describing
  projects in way that it can be reproduced (built) is the fist necessary step
  before introducing CI/CD systems.
```

Reply:
So, let's just fix the typo then? if the problem is not clear. Changing language a bit is fine either if it sounds/reads better, but let's not change the essence if there is no clear understanding what exactly are we fixing.

Reply:
Yes, that's what I meant. The typo is fixed in the suggestion I left. We just need to commit it. Allow me...

Reply:
To me the original version echos reproducibility rather than pipeline flexibility. The piece that matters for CI/CD is DVC can figure out when something important has changed (data, code, etc) and re-run a pipeline correctly. Like what @jorgeorpinel said "pipelines allow CI/CD systems reproduce all or part of the process."

Reply:
Thanks Noah, I'll chat with Ivan and continue this elsewhere if needed, since this has been merged. But feel free to contribute another PR if you prefer 🙂

Reply:
No problem! Again, thanks for all of the feedback

Reply:
yep. I agree with the `pipelines allow CI/CD systems reproduce all or part of the process` part. `on production datasets` and mentioning `train` make it specific.

> The piece that matters for CI/CD is DVC can figure out when something important has changed (data, code, etc) and re-run a pipeline correctly

not sure this it true. Since it's already committed it can force `dvc repro` for example to guarantee that external system gets the same result. There are other examples when we can "abuse" CI/CD to do the training first time- we don't change anything per-se - everything comes with a commit, system runs it first time.

Reply:
Also, I think CML.dev link still make a lot of sense.